src/bugbug/train/requirements.txt

```diff
@@ -1,8 +1,7 @@
 -e ./../../../lib/cli_common[taskcluster,mercurial,log] #egg=mozilla-cli-common
 
-spacy==2.0.12  # pypi2nix fails with newer versions
-thinc==6.10.3  # pypi2nix fails with newer versions
-murmurhash==0.28.0  # pypi2nix fails with newer versions
-pyOpenSSL==18.0.0
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz#egg=en-core-web-sm
+idna<2.8,>=2.5  # Could not find a version that satisfies the requirement idna<2.8,>=2.5 (from requests==2.20.1)
+msgpack<0.6.0,>=0.5.6  # Could not find a version that satisfies the requirement msgpack<0.6.0,>=0.5.6 (from thinc==6.12.1)
+pyOpenSSL>=0.14  # Could not find a version that satisfies the requirement pyOpenSSL>=0.14; extra == "security" (from requests[security]>=2.7.0->libmozdata==0.1.43)
```

Comment:
It's a pity that we have to specify these requirements directly :(

Reply:
@marco-c yeah it is a pity.

but at least we catch this errors with nix. otherwise we would be blindly believe what pip gives us is correct and is far from the truth. the broken part of this process is pip and since it is silent about it nobody complains.

Reply:
Yeah we should try to fix them upstream, but it would take some time...

Reply:
the problem is not how dependencies are defined upstream. the problem here is pip resolver and how it handles requirements.txt